recipes-containers/kubelet/files/set-kubelet-ip.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+# Copyright (c) 2021, Pelion and affiliates.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------
+
+# -------- INSTRUCTIONS --------
+# Change the Pelion kubelet IP and Pod CIDR.
+#
+# 1. NODE_IP sets the kubelet IP by modifying the `kaas0` interface.
+# 2. POD_CIDR and POD_CIDR_GW sets the IP range for Pods. The POD_CIDR_GW must 
+#    correspond to the gateway of the interface created from the POD_CIDR 
+#    subnet. For /24 subnet, this would normally be the IP with the last number 
+#    set to "1".
+# 3. Run the script.
+
+NODE_IP="172.21.1.0"
```

Comment:
The idea of modifying these on the file might be a tad bit difficult, as you might not have rights to modify this file (if it's part of the OSTree managed area, it's in read only mode).  Ideally the setting script should take these as parameters on the command line instead @ZakariasL ?

@kivaisan  @petedyerarm - how do you feel?

Reply:
I guess short term workaround is to advice people to copy this over to some place, where they CAN modify it and then run it. We need a docs topic to pelion-dm-edge-docs. @ZakariasL  @AnotherButler

Reply:
Ah, I assumed it was read/write mode for everyone based on how it was copied to the location: https://github.com/PelionIoT/meta-pelion-edge/blob/74c8fb2bab93b6c622895adb0b5c5210932885bb/recipes-containers/kubelet/kubelet_git.bb#L62

Reply:
My bad, I see now that it only has write permission for the owner of the file. Let me know if I should make it into arguments instead